surge:
  input_cam_front: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
  input_cam_bottom: [1.0, 1.0, 2.0, 1.0, 1.0, 1.0]
  input_imu_pos: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
  input_imu_accel: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
  input_depth: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
sway:
  input_cam_front: [0.028, 0.036, 0.032, 1.0, 1.0, 1.0]
  input_cam_bottom: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
  input_imu_pos: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
  input_imu_accel: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
  input_depth: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
heave:
  input_cam_front: [0.5, 0.3, 0.3, 0.01, 0.01, 0.01]
  input_cam_bottom: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
  input_imu_pos: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
  input_imu_accel: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
  input_depth: [0.6, 0.0, 0.0, 100.0, 100.0, 100.0]
roll:
  input_cam_front: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
  input_cam_bottom: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
  input_imu_pos: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
  input_imu_accel: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
  input_depth: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
pitch:
  input_cam_front: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
  input_cam_bottom: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
  input_imu_pos: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
  input_imu_accel: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
  input_depth: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
yaw:
  input_cam_front: [5.0, 1.5, 3.5, 0.01, 0.01, 0.01]
  input_cam_bottom: [0.5, 1.0, 1.0, 1.0, 1.0, 1.0]
  input_imu_pos: [0.5, 0.0, 0.0, 0.1, 0.1, 0.1]
  input_imu_accel: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
  input_depth: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
